src/open_inwoner/configurations/management/commands/generate_config_docs.py

```diff
@@ -0,0 +1,115 @@
+import os
+from pathlib import Path
+from typing import TypeAlias
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+
+from open_inwoner.configurations.bootstrap.models import (
+    DigiDOIDCConfigurationSettings,
+    KICConfigurationSettings,
+    SiteConfigurationSettings,
+    ZGWConfigurationSettings,
+    eHerkenningDOIDCConfigurationSettings,
+)
+
+ConfigSetting: TypeAlias = (
+    DigiDOIDCConfigurationSettings
+    | KICConfigurationSettings
+    | SiteConfigurationSettings
+    | ZGWConfigurationSettings
+    | eHerkenningDOIDCConfigurationSettings
+)
+
+
+SUPPORTED_OPTIONS = ["siteconfig", "kic", "zgw", "digid_oidc", "eherkenning_oidc"]
+
+TEMPLATE_DIR = (
+    Path(os.path.abspath(os.path.dirname(__file__))).parent.parent
+    / "bootstrap"
+    / "templates"
+)
+TARGET_DIR = Path(settings.BASE_DIR) / "docs" / "configuration"
+
+
+class Command(BaseCommand):
+    help = "Create docs for configuration setup steps"
+
+    def add_arguments(self, parser):
+        parser.add_argument("config_option", nargs="?")
+
+    def get_config(self, config_option: str) -> dict[str, ConfigSetting]:
+        mapping = {
+            "siteconfig": SiteConfigurationSettings,
+            "kic": KICConfigurationSettings,
+            "zgw": ZGWConfigurationSettings,
+            "digid_oidc": DigiDOIDCConfigurationSettings,
+            "eherkenning_oidc": eHerkenningDOIDCConfigurationSettings,
+        }
+        return mapping[config_option]
+
+    def get_detailed_info(self, config: ConfigSetting) -> list[str]:
+        ret = []
+        for field in config.fields["all"]:
+            model_field = config.model._meta.get_field(field.name)
+            part = []
+            part.append(f"{'Variable':<20}{config.get_setting_name(field)}")
+            part.append(f"{'Setting':<20}{str(model_field.verbose_name)}")
+            part.append(f"{'Description':<20}{str(model_field.help_text)}")
```

Comment:
At this point you shouldn't need the model field anymore, the `populate_fields()` earlier was already responsible to collect the meta data (eg: `verbose_name` and `help_text`) and put them in the `ConfigField` instances that would them be the abstraction and source of truth for everything else that follows.